sentence_transformers/util.py

```diff
@@ -621,13 +625,16 @@ def mine_hard_negatives(
     Args:
         dataset (Dataset): A dataset containing (anchor, positive) pairs.
         model (SentenceTransformer): A SentenceTransformer model to use for embedding the sentences.
+        query_column_name (str, optional): The column name in `dataset` that contains the queries. Defaults to None, in which case the first column in `dataset` will be used.
```

Comment:
```suggestion
        anchor_column_name (str, optional): The column name in `dataset` that contains the anchors or queries. Defaults to None, in which case the first column in `dataset` will be used.
```
Could we rename this to `anchor`, fitting with the `anchor, positive, negative` scheme? That way, people don't get the impression that this can only be used for "query-document" information retrieval datasets, but also e.g. for semantic textual similarity (although I imagine 90% of usage is for IR).

Reply:
Thank you.
Would you suggest renaming:
`query_column_name`: to `anchor_column_name`
right?

Reply:
Indeed! I think it looks good now